xbit/routes/index.js

```diff
@@ -45,6 +46,11 @@ router.get('/logout', function(req, res, next) {
   });
 });
 
+router.get('/forgot', userControl.getForgot);
+router.post('/forgot', userControl.postForgot);
+router.get('/reset/:token', userControl.getReset);
+router.post('/reset/:token', userControl.postReset);
+
```

Comment:
if we organize the routes using express.router, we can then simplify this to 
router.use('/foo', require('foo.js'));